UnityProject/Assets/Scripts/Player/Mind.cs

```diff
@@ -52,9 +94,24 @@ public static void Create(GameObject player)
 
 	public void SetNewBody(PlayerScript playerScript)
 	{
+		Spells.Clear();
 		playerScript.mind = this;
 		body = playerScript;
 		bodyMobID = playerScript.GetComponent<LivingHealthBehaviour>().mobID;
+		if (occupation != null)
+		{
+			int i = 0;
+			foreach (var spellData in occupation.Spells)
+			{
+				var spellScript = spellData.AddToPlayer(playerScript);
+				Spells.Add(spellScript);
+			}
+
+			if (occupation.JobType == JobType.MIME)
```

Comment:
I think it'd be better if this was set by the job somehow rather than the mind so conceptually they're separate which will be more maintainable in the future. Job specifics shouldn't be in the domain of the Mind, otherwise you could end up with a long list in here in the future.

You could have a list of properties defined in a job which they set on the body/mind when initialised (similar to the inventory populators for example).

Reply:
yeah, I'll try integrating that into Occupation ScriptableObject